mintlify/mint
# Bump CLI 3.0.16

## Summary
Starting from the lowest level of the dependency tree to the highest:
- Bumps validation in @mintlify/previewing
- Points @mintlify/previewing to newest mint release with openApi -> openapi change
- Bumps @mintlify/cli to use newest @mintlify/previewing
- Published the new versions of @mintlify/previewing and @mintlify/cli to the npm registry

### To-Do
- [ ] Publish `mintlify` to 3.0.16 after this merges.

## Test Plan
- If you have a global installation of @mintlify/cli or mintlify, remove it by running `npm uninstall mintlify -g` or `npm uninstall @mintlify/cli -g`
- Run`npm i @mintlify/cli -g`
- Run `mintlify dev` in a docs with `openapi` in their `mint.json` (You can use [mindsdb](https://github.com/mindsdb/mindsdb))
- To check it's propagating properly navigate to `~/.mintlify/mint/client/src/_props/openApiFiles.json` - there should be content there.

Note - We are essentially using `@mintlify/cli` as a staging env before publishing to prod at `mintlify`
